src/lib.rs

```diff
@@ -16,3 +16,99 @@ pub mod table;
+        // This uses geo-traits to serialize to WKB and adds the binary to the array
+        wkb_builder.push_polygon(Some(&p0()));
+
+        // Finish the builder, creating an array of logical length 1.
+        let wkb_arr = wkb_builder.finish();
+
+        // Access the WKB scalar at position 0
+        // This is a reference onto the array. At this point the WKB is just a "blob" with no other
+        // information.
+        let wkb_scalar = wkb_arr.value(0);
+
+        // This is a "parsed" WKB object. The [WKBGeometry] type is an enum over each geometry
+        // type. WKBGeometry itself implements GeometryTrait but we need to unpack this to a
+        // WKBPolygon to access the object that has the PolygonTrait impl
+        let wkb_object = wkb_scalar.to_wkb_object();
+
+        // This is a WKBPolygon. It's already been scanned to know where each ring starts and ends,
+        // so it's O(1) from this point to access any single coordinate.
+        let wkb_polygon = match wkb_object {
+            WKBGeometry::Polygon(wkb_polygon) => wkb_polygon,
+            _ => unreachable!(),
+        };
+
+        // Add this wkb object directly into the BoundingRect
+        let mut bounding_rect = BoundingRect::new();
+        bounding_rect.add_polygon(&wkb_polygon);
+
+        assert_eq!(bounding_rect.minx, -111.);
+        assert_eq!(bounding_rect.miny, 41.);
+        assert_eq!(bounding_rect.maxx, -104.);
+        assert_eq!(bounding_rect.maxy, 45.);
+    }
+
+    #[test]
+    fn test_wkb_to_frechet() {
+        use crate::algorithm::geo::{
+            FrechetDistance as OtherFrechetDistance, FrechetDistanceLineString,
+        };
+        use crate::test::linestring::{ls0, ls1};
+
+        let mut wkb_builder = WKBBuilder::<i32>::new();
+        wkb_builder.push_line_string(Some(&ls0()));
+        let wkb_arr = wkb_builder.finish();
+        let wkb_scalar = wkb_arr.value(0);
+        let wkb_object = wkb_scalar.to_wkb_object();
+        let wkb_ls0 = match wkb_object {
+            WKBGeometry::LineString(wkb) => wkb,
+            _ => unreachable!(),
+        };
+
+        let mut wkb_builder = WKBBuilder::<i32>::new();
+        wkb_builder.push_line_string(Some(&ls1()));
+        let wkb_arr = wkb_builder.finish();
+        let wkb_scalar = wkb_arr.value(0);
+        let wkb_object = wkb_scalar.to_wkb_object();
+        let wkb_ls1 = match wkb_object {
+            WKBGeometry::LineString(wkb) => wkb,
+            _ => unreachable!(),
+        };
+
+        let expected = 5.656854249492381;
+
+        // passes
+        use crate::io::geo::line_string_to_geo;
+        use geo::FrechetDistance;
+        let distance = line_string_to_geo(&wkb_ls0).frechet_distance(&line_string_to_geo(&wkb_ls1));
+        approx::assert_relative_eq!(distance, expected);
+
+        // DOES NOT COMPILE:
+        //     error[E0599]: no method named `frechet_distance` found for struct `array::binary::array::WKBArray` in the current scope
```

Comment:
I understand the immediate cause of the error, the trait impl doesn't exist. But why doesn't it? I guess I'm saying I don't understand yet how I'm _supposed_ to use the FrechetDistance trait.